src/routes/SSO.mjs

```diff
@@ -26,6 +27,10 @@ export default class SSO extends API {
       },
     })
 
+    if (ctx.state.permissions.includes('rescues.read') === false) {
```

Comment:
Why not just introduce scopes for these so there's no need to piggy back off an unrelated permission.